pkg/version/version.go

```diff
@@ -31,7 +31,7 @@ var (
 	State       string
 
 	// MaxPluginVersion must be increased to add new plugin and make sure the new plugin version is same to this value
-	MaxPluginVersion = 2
+	MaxPluginVersion = 3
```

Comment:
@vburenin In the PR I merged yesterday, I already used version=2, and at that time `rename` feature was not added yet. So containers with dataversion=2 does not support rename. Actually rename on such containers does work; but it is Emma's strong suggestion from her upgrade framework's POV.